### ISSUE TITLE
git help checker is incorrect for Windows; always get error "git command not found, please install it"

### ISSUE THREAD

**Issue by GitMurf** [isakbm/gitgraph.nvim#21]
I think the help utility checker ([see here](https://github.com/isakbm/gitgraph.nvim/blob/c6b6419caf106270b51a1a6f7049b40c81d2b413/lua/gitgraph/helpers.lua#L60)) for git install is incorrect for Windows. The graph never even tries to load because it gets blocked by the git checker with: "git command not found, please install it"

This is the result I get when running the 

`"git version 2.45.2.windows.1"`

The checker seems to expect multiple lines and makes sure the last one is not equal to 0 but it appears on Windows it all just comes back as a single string.

Or am I mis-understanding something on how this is working?

**Comment by isakbm** [comment]
That seems very plausible.

Please try to find a solution and make a PR. I will not be able to resolve this myself as I use WSL and therefore I don't run vim inside windows directly.

I don't even know how to get vim to work correctly and performantly on windows. Every time I tried it was both easier and faster to get it running inside WSL.

**Comment by brunobmello25** [comment]
hey, I believe I managed to fix this issue with [this PR](https://github.com/isakbm/gitgraph.nvim/pull/22)

@GitMurf could you test it in your machine as well and see if it's working as intended? just clone the fork and adjust your plugin manager to pull gitgraph.nvim from your cloned fork. If you are using lazy, you can do this like this:

```lua
  {
    'isakbm/gitgraph.nvim',
    dependencies = {
      'sindrets/diffview.nvim',
    },
    config = function()
      -- ...
    end,
  }
```

**Comment by GitMurf** [comment]
> could you test it in your machine as well and see if it's working as intended? just clone the fork and adjust your plugin manager to pull gitgraph.nvim from your cloned fork.

@brunobmello25 i assume you mean use your fork here? https://github.com/brunobmello25/gitgraph.nvim

**Comment by brunobmello25** [comment]
> > could you test it in your machine as well and see if it's working as intended? just clone the fork and adjust your plugin manager to pull gitgraph.nvim from your cloned fork.
> 
> @brunobmello25 i assume you mean use your fork here? https://github.com/brunobmello25/gitgraph.nvim
> 
> Instead of cloning, can't I just point lazy plugin spec to your repo?

You are right! You can do that to 😅

**Comment by GitMurf** [comment]
@brunobmello25 your PR fixes the git check cmd util function. nice work. But there is one more fix needed for the plugin to work on windows (cmd). Instead of me doing a separate PR, do you mind just adding this fix as well? all you need to do is update this line here: https://github.com/brunobmello25/gitgraph.nvim/blob/9c0fb9290b195d9cc60e9257b193268871b88d8d/lua/gitgraph/git.lua#L25

To this:

```lua
  local cli = [[git log %s %s --pretty="%s" --date="%s" %s %s --date-order]]
```

The change is simple... just needs `"` instead of `'` for quotes. I am not sure if you want to add a `is_windows` condition on this or if it works the same in unix shells (I cannot test).

Let me know your thoughts and if you have any questions?

**Comment by brunobmello25** [comment]
@GitMurf thanks for helping test this out!

I tried running this git log command in a windows shell, but it is still failing:

![image](https://github.com/user-attachments/assets/1c1b076b-03e1-4222-9cde-c2d3c4bfa927)

Trying to figure out here what is making this command ambiguous

**Comment by brunobmello25** [comment]
@GitMurf thanks for testing and helping with the PR! I've just pushed the fix. could you test again? should be working now

**Comment by GitMurf** [comment]
@brunobmello25 confirmed working! nice work and thanks!

**Comment by GitMurf** [comment]
@isakbm and @brunobmello25 just confirming that after the merged PR and updating the plugin everything is working as expected on Windows!
